### PR TITLE
Removed Web Setup Wizard content (install)

### PR DESCRIPTION
**Pull request by jeff-matthews**
## Purpose of this pull request

This pull request (PR) removes references to the Web Setup Wizard as it relates to installation. 

This is the first of two passes on devdocs. It addresses changes related to install only. I will address changes related to upgrade on the second pass because this PR is already so large.

Both passes will be merged with the `remove-wizard-integration` branch before being merged with `2.4.0-develop`.

- Removed 2.4 topics in Setup > Installation Guide > Setup Wizard Install
  - Redirected 2.4 topics to 2.4 CLI install topics
  - Restricted 2.3 topics in the TOC
- Broke symlinks to 2.3 files where deprecation messaging about the wizard must remain for 2.3
  - Created new 2.4 includes for 2.4 files
  - Removed deprecation messaging and references to the wizard in new includes for 2.4 files

I still need to update the [Installation Flow Diagram](https://devdocs.magento.com/guides/v2.3/install-gde/install-resource-diagram.html), but I'm going to need some help with that since we don't have a source file to edit the image and I'm not sure what the design should be.

## Affected DevDocs pages

The primary area of focus should be on the pages inside the Setup Wizard Install section of the Installation Guide. These topics should appear in 2.3, but should redirect to the CLI install topics in 2.4:

Screen Shot 2020-05-19 at 4 54 01 PM

## Additional info

See internal ticket MC-34420 for more info and link to staging build

### CODE REVIEW

**Comment by keharper** [comment]
One more thing: Consider editing the installation YML file so that the "Install optional sample data modules" section so that appears after the "Install the Magento software using the command line" section.

File: src/_includes/install/file-system-perms-before_24.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+## Set pre-installation file system ownership and permissions {#perms-over}
+
+This topic discusses how to set read-write permissions for the web server group before you install the Magento software. This is necessary so the command line can write files to the Magento file system.
+
+The procedure you use is different, depending on whether you use [shared hosting](#perms-shared) and have one user or if you use a [private server](#perms-private) and have two users.
+
+{:.bs-callout-info}
```
[the remark this answers is not in the document]
Comment (jeff-matthews) [reply src/_includes/install/file-system-perms-before_24.md:7]: Will delete 2.4 wizard screenshots added in https://github.com/magento/devdocs/pull/7039

**Comment by keharper** [comment]
I didn't see any issues in or near the recently-changed text.

However, while clicking through each page in the Install Guide, I saw references to the wizard. 

src/_includes/install/file-system-perms-before_24.md
This include file contains two more problematic include files: install/file-system-perms-oneuser_22.md and install/file-system-perms-twouser_22.md. Both of these files mention the Setup wizard. I'm looking at that install/include  directory and wondering if the corresponding files that don't contain _22 in the name are still valid.

I'm seeing the same thing in 
install-gde/prereq/file-system-perms.html
install-gde/prereq/zip_install.html
install-gde/install/prepare-install.htm
install-gde/install/cli/install-cli-uninstall.html
install-gde/install/cli/install-cli-subcommands-deployment.html

**Comment by jeff-matthews** [comment]
Thanks @keharper. I'll do another sweep for references to the Wizard in 2.4 topics. I thought I had gotten all of them.

**Pull request merged**